src/components/Thanks/BadgesCustomization.vue

```diff
@@ -176,6 +198,37 @@ export default {
 		showNewBg() {
 			this.newBgActive = true;
 		},
+		countLoansInCategories() {
+			this.defaultSortBadges.forEach((badge, idx) => {
+				this.loans.forEach(loan => {
+					let matches = true;
```

Comment:
I think `matches` should probably default to `false`

Reply:
Is there a situation where a loan will match none of the conditions? In that case, the default would be `true`, which would be incorrect, right? Maybe I'm reading the logic wrong.

Reply:
Ah got it, so we'll never currently encounter a situation where we'd get `true` to fall through accidentally?